frontend/javascripts/oxalis/model/reducers/volumetracing_reducer.ts

```diff
@@ -305,6 +318,74 @@ function getVolumeTracingFromAction(state: OxalisState, action: VolumeTracingRed
   return getVolumeTracingById(state.annotation, maybeVolumeLayer.tracingId);
 }
 
+export function toggleSegmentGroupReducer(
```

Comment:
these two toggle functions are similar to the tree counterparts. however, I found them cumbersome to unify.